mflike/theoryforge.py

```diff
@@ -16,9 +16,8 @@ def _cmb2bb(nu):
     return np.exp(x) * (nu * x / np.expm1(x)) ** 2
 
 
-class TheoryForge_MFLike:
+class TheoryForge:
```

Comment:
Hi @xgarrido, the name change is ok. We could think to keep the name `TheoryForge_MFLike` only in `SOLikeT`, where a more explicit reference to mflike can be useful

Reply:
I agree with you that's why I changed